src/api.ts

```diff
@@ -170,6 +170,7 @@ interface BaseParameter {
  */
 export interface StringParameter extends BaseParameter {
     type: "string"
+    /** Regex that must match the expected string. Provide a regex starting with ^ and ending with $ so that it matches the whole string. */
     regex?: string //we can't use RegExp here since it can't be exported to JSON and not be exported by GoJa
```

Comment:
Wouldn't it be better to join the two comments?
I mean something like:

```
/** Regex that must match the expected string. Provide a regex starting with ^ and ending with $ so that it matches the whole string. 
We can't use RegExp here since it can't be exported to JSON and not be exported by GoJa.
*/
```